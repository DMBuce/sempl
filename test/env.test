#!/usr/bin/env sempl
# check runtime environment of the tests
SEMPL_BRACES={{}}
#!end sempl env

Check that $PATH is set correctly
PATH="{{!grep -o '^\./bin:' <<< "$PATH"}}$PATH"

Check that we're using the right `sempl`
{!which sempl | sed "s,^$(git rev-parse --show-toplevel),\$gitroot,"}:
  =>{{!which sempl | sed "s,$(git rev-parse --show-toplevel),\$gitroot,"}}<=

