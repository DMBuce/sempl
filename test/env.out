
Check that $PATH is set correctly
PATH="./bin:$PATH"

Check that we're using the right `sempl`
{!which sempl | sed "s,^$(git rev-parse --show-toplevel),\$gitroot,"}:
  =>$gitroot/bin/sempl<=

